docs/en/setup/plugins/queue_mmap-queue.md

```diff
@@ -4,17 +4,15 @@ This is a memory mapped queue to provide the persistent storage for the input ev
 ## DefaultConfig
 ```yaml
 # The size of each segment. Default value is 128K. The unit is Byte.
-segment_size: 131072
+segment_size: 262114
```

Comment:
do you mean `262144`? `262144 == 256K` while `262114` is odd. Also need to update the comment above (line 6)